openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -393,11 +393,12 @@ The response from the Credential Issuer that contains a Credential Offer Object
 
 This ability to pass the Credential Offer by reference is particularly useful for large requests.
 
-Below is a non-normative example of the Credential Offer displayed by the Credential Issuer as a QR code when the Credential Offer is passed by reference:
+When the Credential Offer is passed by reference, the Credential Offer can be displayed by the Credential Issuer as a QR code. Below is a non-normative example:
```

Comment:
One could argue that the example is not a QR code and that this QR codes may also be used without passing by reference

Reply:
```suggestion
The Credential Offer can be displayed by the Credential Issuer as a QR code. Below is a non-normative example of a content of such a QR code when the Credential Offer is passed by reference:
```

Reply:
```suggestion
When the Credential Offer is displayed as a QR code, it would usually contain the Credential Offer by reference due to the size limitations of the QR codes. Below is a non-normative example:
```

Anyone can render the QR code